webhook_to_fedora_messaging/endpoints/util.py

```diff
@@ -0,0 +1,60 @@
+from flask import Response, Request
+from ..config.defaults import FASJSON_URL
+import fasjson_client
+
+
+def not_found() -> Response:
+    return Response({'message': 'Not Found'}, status=404, mimetype='application/json')
+
+
+def success(data: dict) ->Response:
+    return Response(data, status=200, mimetype='application/json')
+
+
+def bad_request() -> Response:
+    return Response("{'message': 'Bad Request'}", status=400, mimetype='application/json')
```

Comment:
This is equivalent to returning `{'message': 'Bad Request'}`, 400. Please see the doc above to setup a JSON-aware error handler that will allow us to just raise exceptions or use `flask.abort()`.